src/multi_vector_simulator/utils/__init__.py

```diff
@@ -170,3 +172,29 @@ def compare_input_parameters_with_reference(folder_path, ext=JSON_EXT):
         answer[EXTRA_PARAMETERS_KEY] = extra_parameters
 
     return answer
+
+
+def copy_report_assets(path_destination_folder):
+    """Copy the css file and eland logo to the path_destination_folder
+
+    Parameters
+    ----------
+    path_destination_folder
```

Comment:
Is `path_destination_folder` a string, what does it do?

Reply:
Fixed in 6c492de53d79dce187b35df3cdecd5b30c4fb077